free-programming-books.md

```diff
@@ -1137,7 +1136,7 @@ Original Source: [List of freely available programming books](http://web.archive
 * [Introduction to Haskell](http://www.seas.upenn.edu/~cis194/spring13/) - Brent Yorgey
 * [Learn Haskell Fast and Hard](http://yannesposito.com/Scratch/en/blog/Haskell-the-Hard-Way/)
 * [Learn You a Haskell for Great Good](http://learnyouahaskell.com) - Miran Lipovaca
-* [Natural Language Processing for the Working Programmer](http://nlpwp.org/book/)
```

Comment:
1. The main page says that the book is unfinished, and is unlikely to be revisited by its authors. Should we even keep such a book on the list?
2. I think the fact the website was taken down shows disinterest by the authors -- another reason to take the book off the list.